03week/towersOfHanoi.js

```diff
@@ -35,6 +41,7 @@ const isLegal=(startStack,endStack)=> {
     }
 }
 }
+//   this function is used to determine which blocks become and endStack or a StartStack in this case there is only 3 possible outcome we have 3 stacks which are the 3 areas where the items will be place . if a is a start stack b and c naturally become end stack same for b"startStack where a and c are now end stacks and when c is a startStack  then naturally a and b now become end stacks
```

Comment:
fix the indentation here and this function can be greatly simplified, you are repeating yourself a lot and you can just return the evaluation